tools/enso4igv/package.json

```diff
@@ -77,19 +77,19 @@
     "lint": "tslint -p ./"
   },
   "dependencies": {
-    "copy-webpack-plugin": "^10.2.4",
-    "decompress": "4.2.1",
-    "maven": "^5.0.0"
+    "copy-webpack-plugin": "=10.2.4",
+    "decompress": "=4.2.1",
+    "maven": "=5.0.0"
   },
   "devDependencies": {
     "@types/vscode": "=1.49.0",
-    "@types/xml2js": "^0.4.8",
-    "ts-loader": "^6.2.1",
-    "tslint": "^6.1.0",
-    "typescript": "^3.9.10",
-    "vsce": "^2.6.4",
-    "webpack": "^5.40.0",
-    "webpack-cli": "^4.7.2"
+    "@types/xml2js": "=0.4.11",
```

Comment:
Using `^` allows the node to download new version of the same library. That's often dangerous as new versions may not be fully compatible with previous ones. Using `=` to fix to an exact version of each library. This configuration is known to work.